use-cases/local-rag.ipynb

```diff
@@ -67,7 +67,7 @@
     "from pymongo import MongoClient\n",
     "from sentence_transformers import SentenceTransformer\n",
     "\n",
-    "# Connect to your local Atlas deployment or Atlas Cluster\n",
+    "# Connect to your local Atlas deployment or MongoDB Cluster\n",
```

Comment:
I'm confused...? so it's a local Atlas deployment _or_ a MongoDB cluster?
```suggestion
    "# Connect to your local MongoDB deployment or MongoDB cluster\n",
```

Reply:
Yeah... MongoDB cluster refers to both local (on-prem) and cloud (Atlas). local Atlas deployment is a special local deployment of Atlas.

Just using the terminology guidance that I was given ...